### PR TITLE
[DOCFIX] Fix some descriptions related to the FileSystemMaster module

### PR DESCRIPTION
**Pull request by jianghuazhu**
### What changes are proposed in this pull request?
The purpose of this pr is to fix some descriptions related to the FileSystemMaster module.

### Why are the changes needed?
In the FileSystemMaster module, there are some inappropriate descriptions or comments, and we should fix them as much as possible.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

**Comment by jianghuazhu** [comment]
Can you help review this PR, @dbw9580  @JiamingMai .
Thanks.

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 1]
```diff
@@ -397,6 +397,7 @@ public class DefaultFileSystemMaster extends CoreMaster
   /** Thread pool which asynchronously handles the completion of persist jobs. */
   private java.util.concurrent.ThreadPoolExecutor mPersistCheckerPool;
 
+  /** Manager for the Active UFS sync process. */
```
Comment (ssz1997) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:400]: The comment you add here is literally the name of the class, `ActiveSyncManager`, so I don't see a need to add this comment.

**Review by ssz1997 — COMMENTED** [review]
Just one small comment.

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java (hunk 1 above)
Comment (jianghuazhu) [reply core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:400]: Thanks for your comment, @ssz1997 .
I'll delete this line later.

**Comment by jianghuazhu** [comment]
Can you review this PR again, @ssz1997 .
Thanks.

**Review by ssz1997 — APPROVED** [review]

**Comment by jianghuazhu** [comment]
Thanks @ssz1997 .

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks for the work!

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**